python/python/tests/test_async_client.py

```diff
@@ -3203,6 +3204,120 @@ async def test_type(self, redis_client: TRedisClient):
 
         assert (await redis_client.type(key)).lower() == "none"
 
+    @pytest.mark.parametrize("cluster_mode", [False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_sort_and_sort_store_with_get_or_by_args(
+        self, redis_client: TRedisClient
```

Comment:
I think you can do
```suggestion
        self, redis_client: RedisClient
```
and remove `if` with `isinstance` on line 3230